app/javascript/components/froggo-checkbox.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="flex w-full p-2">
+    <input
+      :id="id"
+      type="checkbox"
+      class="hidden"
+      v-model="model"
+      v-bind="$attrs"
+      :value="inputValue"
+    >
+    <label
+      :for="id"
+      class="self-center block w-4 h-4 mr-4 text-white duration-100 ease-in-out bg-gray-400 rounded cursor-pointer transition-color"
+      :class="value.includes(inputValue) && 'bg-froggoGreen-500'"
+    >
+      <inline-svg
+        :src="require('../../assets/images/icons/check-icon.svg').default"
+        class="w-full h-full transition-opacity duration-100 ease-in-out opacity-0 fill-current"
+        :class="value.includes(inputValue) && 'opacity-100'"
+      />
+
+    </label>
+    <label
+      v-if="label"
+      :for="id"
+    >
+      <img
+        :src="avatar"
+        class="h-10 mr-4"
+      >
+    </label>
+    <label
+      v-if="label"
+      :for="id"
+      class="self-center text-base cursor-pointer select-none"
+    >
+      {{ label }}
+    </label>
+    <label
+      v-if="label"
+      :for="id"
+      class="self-center ml-auto text-sm text-gray-400 cursor-pointer select-none"
+    >
+      {{ capitalizeRole }}
+    </label>
```

Comment:
Acá quizás podría ir todo metido en un `label` con el `v-if` y el `:for`?

```suggestion
    <label
      v-if="label"
      :for="id"
      class="flex items-center"
    >
      <img
        :src="avatar"
        class="h-10 mr-4"
      >
      <span class="text-base cursor-pointer select-none">
        {{ label }}
      </label>
      <span class="ml-auto text-sm text-gray-400 cursor-pointer select-none">
        {{ capitalizeRole }}
      </span>
    </label>
```

Cambié un par de clases para que se preserve como se ve, pero creo que así es más "limpio", tipo que si `label` existe entonces se muestra la agrupación de elementos